src/_styles/_socialMedia.scss

```diff
@@ -0,0 +1,18 @@
+.social-media {
+  display: flex;
+  .icon {

```

Comment:
Let's add `width` and `height` params for icon.

Reply:
- [ ] Done [501c15b](https://github.com/AnastasiSo/Portfolio-Project/pull/12/commits/501c15be0a2154e0f0aca6168a64c62f97c90f28)